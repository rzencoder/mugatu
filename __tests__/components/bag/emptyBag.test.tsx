import React from 'react'
import { render } from '@testing-library/react'
import EmptyBag from '@/components/bag/emptyBag'

describe('<EmptyBag />', () => {
  it('renders <EmptyBag /> component', () => {
    const { container } = render(<EmptyBag />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
