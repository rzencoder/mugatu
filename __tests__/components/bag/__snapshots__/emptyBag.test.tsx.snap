// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EmptyBag /> renders <EmptyBag /> component 1`] = `
<div
  class="css-1u27tmr"
>
  <div
    class="css-1mgati5"
  >
    Your Shopping Bag Is Empty
  </div>
  <a
    class="chakra-link css-0"
    href="/"
  >
    <button
      class="chakra-button css-oqfhsh"
      type="button"
    >
      Continue Shopping
    </button>
  </a>
</div>
`;
